src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs

```diff
@@ -748,7 +748,7 @@ public async Task<HttpResponseMessage> SendAsyncCore(HttpRequestMessage request,
                         // Map the exception the same way as we normally do.
                         catch (Exception ex) when (MapSendException(ex, cancellationToken, out Exception mappedEx))
                         {
-                            throw mappedEx;
+                            throw;
```

Comment:
This seems unrelated... and like it's probably not desired.

Reply:
oh yeah it is caught because of a new analyzer [CA2200](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2200?view=vs-2019), but the update might be not desired

Reply:
Hm, that seems (to me) like a bug in that analyzer, since the thing being thrown isn't the exception variable, but a variable introduced from the exception filter call.  It's not doing the thing the analyzer says not to do ("throw ex;")

Reply:
I was not sure if it will be smart enough to map to the `mappedEx` further, most likely not, with your comment I confirmed that it only maps to the original exception `ex`, then it is definitely the analzyer bug, i guess in this case the analyzer shouldn't warn, I will file an issue for the analzyer